src/layer/index.js

```diff
@@ -83,6 +84,8 @@ export function build(locales) {
 
     lyrFerry.ferry,
 
+    lyrHillshade.hillshading,
```

Comment:
The Terrarium tiles are a bit sensitive to artificial structures like dams that look a bit strange over water, like the Lock and Dam Number 12 on the Mississippi River at Bellevue, Iowa:

[<img src="https://github.com/user-attachments/assets/d14dd440-07bb-47e1-afd7-bbfe0c1f32e4" width="640" alt="A big embankment formed by the hillshading layer in the middle of the river.">](https://preview.ourmap.us/pr/1157/#map=13/42.26072/-90.4113)

If we move this layer lower in the stack, such as right below `water_line`, then the waterbody layer would tend to obscure this and other anomalies over water:

<img src="https://github.com/user-attachments/assets/6a934223-8f1e-478d-a4fb-4247a80d12e9" width="640" alt="Most of the embankment gone, save for a small secondary protrusion on the landward side.">

However, the tradeoff is that the “hill” shading would no longer double as a bathymetry layer at sea:

Before | After
----|----
[<img src="https://github.com/user-attachments/assets/a35e7da4-4b9b-4d3e-b750-c38e4d1f117f" width="400" alt="Seamounts and ridges on the sea floor around the Bahamas.">](https://preview.ourmap.us/pr/1157/#map=5/25.5/-78.06) | <img src="https://github.com/user-attachments/assets/26b19f83-123a-4183-a0b2-ee3073f37926" width="400" alt="Solid water fill.">

While bathymetry is pretty cool, I think it’s far enough out of scope of a (land) transportation map as to be a distraction. It also looks a bit discolored without hypsometric tinting, whereas on land the absence of that tinting is quite reasonable on a non-topographic map. (General-purpose overview maps would prioritize showing the continental shelf using hypsometric tinting before bothering with elevation shading.)

Reply:
Incidentally, #995 gives tunnels a different treatment that wouldn’t really exhibit the cute underlapping effect anyways.

The flash of bathymetry feels like an unsubtle Easter egg, of which we have too few in this project. 🙃 In seriousness, maybe we could load the DEM source and hillshade layer only after the main sources and layers load, to avoid jank and unnecessary bandwidth and power usage. To avoid distracting the user on land, we could fade in the hillshade layer with a `hillshade-exaggeration-transition` property, akin to deferred 3D terrain in F4Map, though I don’t know how well that would perform.